bincopy.py

```diff
@@ -1251,7 +1364,7 @@ def _convert_output_format_type(value):
     fmt = items[0]
     args = tuple()
 
-    if fmt in ['srec', 'ihex']:
+    if fmt in ['srec', 'ihex', "ti_txt"]:
```

Comment:
Replace with`'ti_txt'`. I know, `'` and `"` are both used throughout the code base, but the goal is to use `'` where possible.